src/client/common/external-service-data.js

```diff
@@ -1,28 +1,20 @@
-module.exports = [
-  {database:'BioGrid', url:'http://identifiers.org/biogrid/', search:''},
-  {database:'DrugBank', url:'https://www.drugbank.ca/', search:''},
-  {database:'mirtarBase', url:'http://identifiers.org/mirtarbase/', search:''},
-  {database:'NetPath', url:'http://www.netpath.org/', search:'molecule?molecule_id='},
-  {database:'Panther', url:'http://pantherdb.org/', search:'genes/geneList.do?searchType=basic&fieldName=all&organism=all&listType=1&fieldValue='},
-  {database:'PID', url:null},
-  {database:'PhosphoSitePlus', url:null},
+const { IDENTIFIERS_URL } = require('../../config');
```

Comment:
this file should also be renamed to something related to 'identifiers', not the general 'external-services-data'